pkg/helpers/values.go

```diff
@@ -19,3 +23,16 @@ func GetDeletionPolicy() synv1alpha1.DeletionPolicy {
 		return synv1alpha1.ArchivePolicy
 	}
 }
+
+// RenderTemplate renders a given template with the given data
+func RenderTemplate(tmpl string, data interface{}) (string, error) {
```

Comment:
It might be worth to gift wrap the errors. Doing so, would allow to hint that in one case, the templates syntax is amiss. In the other case, it is probably accessing data not available.

This results in better error messages for the end user. Better in this case would be: easier to understand what is wrong.